bot/utils/regex.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+INVITE_RE = re.compile(
```

Comment:
Liking the initiative to move more stuff to util files! There are certainly a few more expressions I can think of putting in here later.